src/__test__/components/Article.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Article, { Interest } from '../../components/Articles/Article';
+
+describe('App', () => {
+  it('should match snapshot', () => {
+    const wrapper = shallow(<Article />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('it should render interest', () => {
+    const props = {
+      klass: 'likes',
+      alternative: 'likes',
+      src: 'http://myimage.jpg',
+      value: 5,
+    };
+    shallow(<Interest {...props} />);
+  });
+
+  it('it should render interest', () => {
+    const props = {
+      klass: 'dislikes',
+      alternative: 'likes',
+      src: 'http://myimage.jpg',
+      value: 5,
+    };
+    shallow(<Interest {...props} />);
+  });
```

Comment:
You should expect or assert something in all your tests. Please ensure that all your tests have an expectation or assertion

Reply:
sure will update the indentation.

Reply:
Resolved